utils/httpTransport.ts

```diff
@@ -0,0 +1,65 @@
+enum METHODS {
+    GET = 'GET',
+    PUT = 'PUT',
+    POST = 'POST',
+    DELETE = 'DELETE'
+};
+
+function queryStringify(data : object) {
+    let str = '?';
+    
+    for (const [key, value] of Object.entries(data)) {
+        str += key + "=" + value + "&";
+    }
+    str = str.slice(0, str.length - 1)
+    
+    return str
+}
+
+export default class HTTPTransport {
+
+    get = (url : string, options : Object, timeout : number) => {
```

Comment:
```suggestion
    get = (url : string, options : object, timeout : number) => {
```
Было бы здорово определить более точный тип для оptions, указать какие свойства можно передавать data, headers